src/modules/producer.js

```diff
@@ -13,33 +28,43 @@ function createRpcQueue(queue) {
   if (!amqpRPCQueues[queue]) {
     amqpRPCQueues[queue] = {};
   }
-
-  let rpcQueue = amqpRPCQueues[queue];
+  
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.dialonce.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)